src/nft/nft.service.ts

```diff
@@ -10,28 +10,47 @@ export class NftService {
     private prismaService: PrismaService,
   ) {}
 
+  async getById(id: string) {
+    return await this.prismaService.nft.findUnique({
+      where: { id },
+      include: { user: { select: { name: true } } },
+    });
+  }
+
   async getAll() {
     return await this.prismaService.nft.findMany({
-      include: { user: { select: { name: true } } },
+      select: {
+        id: true,
+        title: true,
+        url: true,
+        user: { select: { name: true } },
+      },
```

Comment:
You can create global variable like this:
```ts
const NFT_INCLUDE = Prisma.validator<Prisma.NftDefaultArgs>()({
    select: { id: true, title: true ... },
  })
```

And you can reuse this in different methods (`getAll`, `getById`)